packages/client/src/components/app/forms/Field.svelte

```diff
@@ -55,16 +59,23 @@
   const updateDisabled = disabled => {
     fieldApi?.setDisabled(disabled)
   }
+
+  const updateLabel = e => {
+    builderStore.actions.updateProp("label", e.target.textContent)
+  }
 </script>
 
 <FieldGroupFallback>
   <div class="spectrum-Form-item" use:styleable={$component.styles}>
     <label
+      bind:this={labelNode}
+      contenteditable={$component.editing}
+      on:blur={$component.editing ? updateLabel : null}
       class:hidden={!label}
       for={fieldState?.fieldId}
       class={`spectrum-FieldLabel spectrum-FieldLabel--sizeM spectrum-Form-itemLabel ${labelClass}`}
     >
-      {label || ""}
+      {label || " "}
```

Comment:
Just wondering, why the `" "` - not sure what that does?

Reply:
Haha well spotted. If there is no text then it basically doesn't take up any space, which screws up the table layout that fields use. It means that whenever you double click to start editing text in the preview, the label is misaligned and right up in top left of the field rather than centered. Making sure there is always at least a space seems to fix this so that's why I added it.

Reply:
Makes sense!